### PR TITLE
[ticket/14459] Check language input for group and fix template logic

### PR DESCRIPTION
**Pull request by rubencm**
Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Replaces #4198 

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-14459

### CODE REVIEW

**Comment by 3D-I** [comment]
So the language object it is used (added) in this file just for this specific language stuff while in the rest of it is the user object.

**Comment by rubencm** [comment]
That can be done in other pull request, make a lot of changes in the file can cause conflicts in other pull requests

**Comment by 3D-I** [comment]
So why not to use `$language = $this->phpbb_container->get('language');` only and beforewhere it is needed like in the rest of the file? Then `$language->lang...`

**Comment by rubencm** [comment]
Because it will have to be injected with dependency injection, so is better to leave the variable in the constructor

**Comment by rubencm** [comment]
@3D-I

**Comment by 3D-I** [comment]
I'll take a look at it tonight. Thanks.

**Comment by 3D-I** [comment]
I posted everything at discord ;)

**Comment by 3D-I** [comment]
immagine

**Comment by 3D-I** [comment]
Also, a language key should be removed from the language file now

`    'NO_EXT_GROUP'                => 'None',`

File: phpBB/adm/style/acp_attachments.html [hunk 1]
```diff
@@ -421,17 +421,25 @@ <h2>{L_UPLOADING_FILES}</h2>
 	</tr>
 	</thead>
 	<tbody>
-	<!-- BEGIN attachments -->
+	{% for attachments in attachments %}
```
Comment (rubencm) [inline phpBB/adm/style/acp_attachments.html]: @3D-I do this work? for attachments in attachments

File: phpBB/includes/acp/acp_attachments.php [hunk 2]
```diff
@@ -27,6 +27,9 @@ class acp_attachments
 	/** @var \phpbb\config\config */
 	protected $config;
 
+	/** @var \phpbb\language\language */
+	protected $lang;
```
Comment (marc1706) [inline phpBB/includes/acp/acp_attachments.php]: Please use `$language` as that is what we usually use.

**Pull request merged**